### DISCUSSION TITLE
Is it possible to set a general base load of e.g. 5W?

### DISCUSSION THREAD

**Discussion by Robus007** [syssi/esphome-soyosource-gtn-virtual-meter#193] [Q&A]
Reason: I have an off-grid solar inverter behind the inverter and therefore only a small remaining base load. When the solar inverter is in operation, the normal inverter always fluctuates between 0 and 5 W regulation. That's why I want it to always run at least 5W.

(Sent via google translate)

**Comment by syssi** [comment]
```
soyosource_virtual_meter:
  buffer: -10
```

should do the job!
